requirements/ci32_requirements.txt

```diff
@@ -1,3 +1,4 @@
 spin==0.13
 # Keep this in sync with ci_requirements.txt
-scipy-openblas32==0.3.29.0.0
+scipy-openblas32==0.3.29.0.0 ; sys_platform != 'win32' or platform_machine != 'ARM64'
+scipy-openblas32==0.3.29.265.0 ; sys_platform == 'win32' and platform_machine == 'ARM64'
```

Comment:
Can't easily verify but: this is actually upper-case? If so, that's a pretty poor choice upstream, given that on macOS it's lower-case `arm64`.

Reply:
yes, the upper case is required. Without it, the clause was not respected and the build failed when it tried to install scipy-openblas64 on win-arm64